src/dropdown/dropdown.spec.ts

```diff
@@ -274,6 +274,28 @@ describe('ngb-dropdown-toggle', () => {
          expect(dropdownEl).toHaveCssClass('open');
        });
      })));
+  it('should have css class dropdown on div and dropdown-toggle on button',
```

Comment:
What I was saying is that we can test the functionality in question with the following markup only: `<div ngbDropdown></div>`